app/javascript/article-form/components/EditorActions.jsx

```diff
@@ -31,17 +36,21 @@ export const EditorActions = ({
 
   return (
     <div className="crayons-article-form__footer">
-      <Button className="mr-2 whitespace-nowrap" onClick={onPublish}>
+      <Button
+        variant="primary"
+        className="mr-2 whitespace-nowrap"
+        onClick={onPublish}
+      >
         {published || isVersion1 ? 'Save changes' : 'Publish'}
       </Button>
 
       {!(published || isVersion1) && (
         <Button
-          variant="secondary"
           className="mr-2 whitespace-nowrap"
           onClick={onSaveDraft}
+          title="Save draft"
```

Comment:
Same here, but not blocking either.

Reply:
I know and I don't like it either. The thing is we don't really have enough horizontal space to fit everything on mobile screens :/. I'm actually going to revert that one change where I removed part of this label for mobile because I have some updated mockups for `/new` but I just need time to ship these.